### PR TITLE
nolo tlc ciel d9 9_1b to 9_1_2

### PR DESCRIPTION
**Pull request by nolo16**
arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me. arach is not real, arach cannot hurt me.

### CODE REVIEW

File: script/Ciel/Day 9/09_01_CIEL09_1_2.txt [hunk 1]
```diff
@@ -6,18 +6,17 @@
...
 -- 「え、ホント？　こっち、使っていいの？　こんなノンケの美少年に？
-// 'Who doesn't take it that way' maybe? Unless not clear enough
-"Huh, really? You want me to use this one? You, a young dashing man who isn't swinging that way?
+"Huh, really? You want me to use this one? On you, a young dashing man who doesn't swing that way?
 }
 [sha:77f7b48e3a0c4b40788f28f9530bf9df57842882]{
 -- Page 79, Offset 22337. Mods: @n.
 -- うわやば、アタシもってる。間違いなく蠍座の女の<星辰|うんせい>きちゃってる。
-Oh gawds, I'm in luck. I am blessed by my fortune as a Scorpio woman!
+Oh gawds, I'm in luck. This must be my fortune as a Scorpio woman!
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_1_2.txt:19]: "Oh gawds"??? Just... no. "My god, I'm in luck" or "I'm so lucky" or something. Anything but "oh gawds"

File: script/Ciel/Day 9/09_01_CIEL09_1_2.txt [hunk 2]
```diff
@@ -98,17 +97,17 @@ She then stores away the strange looking and certainly not single-use syringe in
 [sha:fc42b22b9b1b3a52714dc31c83d0ddad5f3e2961]{
 -- Page 82, Offset 22353.
 -- あまりの痛みに腰抜けちゃった？　志貴チャンのバージン奪っちゃった？
-Did you stunned by the pain? Did you lose your virginity?
+Were you stunned by the pain? Did you lose your virginity?
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_1_2.txt:100]: "lose your virginity" is so soulless for バージン奪っちゃった. "Did your virginity get snatched away?"

File: script/Ciel/Day 9/09_01_CIEL09_1_2.txt [hunk 3]
```diff
@@ -98,17 +97,17 @@ She then stores away the strange looking and certainly not single-use syringe in
...
 -- Page 82, Offset 22354.
 -- よちよち、ひとりで立てまちぇんかー？　ならそこのベッドまで連れて行ってあげまちゅねー？
-Good good, can you stand up by yourself? Should I help you to the bed right here?
+Good good, can you stand up by yourself? Should I help you to that bed over there?
 }
 [sha:b29f764a63e00a1a515491834574241c121e70a3]{
 -- Page 82, Offset 22355. Mods: @k, @e.
 -- その後はぁ……悪いところを取り出す手術とかやっちゃう？
-After that... What about a surgery to remove your bad parts?
+After that... How about a surgery to extract your bad parts?
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_1_2.txt:110]: some surgery

File: script/Ciel/Day 9/09_01_CIEL09_1_2.txt [hunk 4]
```diff
@@ -143,27 +142,27 @@ I was bracing for it, but the pain wasn't anything special.
...
 -- Page 84, Offset 22363.
 -- 「うそぅ！？　あ、ホントだ、脈も瞳孔も異常なし！？
-"No way!? Ah, it is true! Your pulse and pupils are in normal condition!?
+"No way!? Ah, you're right! Your pulse and pupils are in normal condition!?
 }
 [sha:51fe73ca09d1fe6a5bcadf6a1d88ae49edbc9688]{
 -- Page 84, Offset 22364.
 -- なーんてあったりまえだっつーの！　だって今のインフルエンザの予防接種じゃなくて、ただの栄養剤だから！　アタシ特製のビタミン剤を食らうといいのだわー！」
-Just kidding, that's totally normal! After all, that wasn't a flu shot, it was just a nutrient injection! It's my specially made vitamin complement!"
+Just kidding, that's totally normal! After all, that wasn't a flu shot, it was just a nutrient injection! It's my specially made vitamin supplement!"
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_1_2.txt:155]: bull shark testosterone

**Review by Avuxo — APPROVED** [review]
Arach is real. Arach can hurt you. Arach is real. Arach can hurt you. Arach is real. Arach can hurt you. Arach is real. Arach can hurt you. Arach is real. Arach can hurt you. 
image

File: script/Ciel/Day 9/09_01_CIEL09_1B.txt [hunk 5]
```diff
@@ -46,37 +46,37 @@ Dashing up the stairs, I attempt the barbaric act of sliding my hands onto Senpa
...
 -- 一方、俺は触れる事すらできず、勢いのまま壁に激突した。
-On the other hand, I'm forcefully flung to the wall without even getting to brush her.
+On the other hand, I'm forcefully flung to the wall, unable to even lay a single finger on her.
 }
 [sha:e8b23700e4ab8dee75c3e6866e6c86f0677e1484]{
 -- Page 44, Offset 22245.
 -- 「と、遠野くんでしたか。
-"O-Oh, Tohno-kun?
+"O-Oh, Tohno-
+-- 「いえ、別に。朝から先輩の顔をkun?
```
[the remark this answers is not in the document]
Comment (nolo16) [reply script/Ciel/Day 9/09_01_CIEL09_1B.txt:60]: jesus christ how did i even do this

**Pull request merged**